package.json

```
@@ -60,14 +61,14 @@
     "immutable": "^4.3.5",
     "jest": "^29.7.0",
     "lockfile-lint": "^4.12.1",
-    "redux": "^4.2.1",
+    "redux": "^5.0.1",
     "redux-immutable": "^4.0.0",
     "rimraf": "^5.0.5",
     "semantic-release": "^22.0.12"
   },
   "peerDependencies": {
     "immutable": "^3||^4.0.0-rc",
-    "redux": "^3||^4",
+    "redux": "^3||^4||^5",
```

```suggestion
    "redux": "<=5",
```
Might make this more maintainable in the future?